controllers/storagecluster/storageclasses.go

```diff
@@ -88,7 +88,7 @@ func (obj *ocsStorageClass) ensureDeleted(r *StorageClusterReconciler, instance
 	return reconcile.Result{}, nil
 }
 
-func (r *StorageClusterReconciler) createStorageClasses(sccs []StorageClassConfiguration) error {
+func (r *StorageClusterReconciler) createStorageClasses(sccs []StorageClassConfiguration, radosNamspaceName string) error {
```

Comment:
i think we dont need to pass the radosNamepsace name as a function parameter, just use the const name as it used for other resouces

Reply:
If it's not needed now, I am in favor of using constant rather than changing function signature. We can always change it later.

Reply:
updated